.github/workflows/pr-open.yml

```diff
@@ -82,14 +82,15 @@ jobs:
             overwrite: true
             parameters: -p MIN_REPLICAS=1 -p MAX_REPLICAS=2 -p BUILD=snapshot-${{ github.event.number }}
     steps:
-      - uses: bcgov-nr/action-deployer-openshift@v0.1.0
+      - uses: bcgov-nr/action-deployer-openshift@v1.0.0
         with:
           file: ${{ matrix.file }}
           oc_namespace: ${{ secrets.OC_NAMESPACE }}
           oc_server: ${{ secrets.OC_SERVER }}
           oc_token: ${{ secrets.OC_TOKEN }}
           overwrite: ${{ matrix.overwrite }}
           penetration_test: false
+          health_url: "/check"
```

Comment:
remove the `/` please. The action can/will fail if it contains that leading `/`. 

@DerekRoberts can you please look into that? The base path concatenation already contains an ending `/` and double `/` generates an issue for curl/wget

Reply:
Please use `verification_path` instead of `health_url`.  Extra slashes ('`/') are ignored by curl in this Action.

Reply:
@DerekRoberts isn't **verification_path**  just for versions above 1.0.0?

Reply:
@paulushcgcj That's right.  Any older and there's no equivalent command.  We had an RC that was different.

Reply:
Correction: Double slashes did cause a problem.  Please update Action to v1.0.1 for the fix.